src/ReactiveUI.AndroidSupport/ControlFetcherMixin.cs

```diff
@@ -92,17 +94,18 @@ public static void WireUpControls(this global::Android.Support.V4.App.Fragment T
 		}
 
 		/// <summary>
-		///
+		/// A helper method to automatically resolve properties in an <see cref="AppCompatActivity"/> to their respective elements in the layout.
 		/// </summary>
-		public static void WireUpControls(this AppCompatActivity This)
+		/// <param name="This"></param>
+		/// <param name="resolveMembers">The strategy used to resolve properties that either subclass <see cref="View"/>, have a <see cref="WireUpResourceAttribute"/> or have a <see cref="IgnoreResourceAttribute"/></param>
+		public static void WireUpControls(this AppCompatActivity This, ResolveStrategy resolveMembers = ResolveStrategy.Implicit)
```

Comment:
I don't think this method adds anything that wouldn't already work with the extension method on the regular `Activity`, since that is what `AppCompatActivity` extends from.

Reply:
I would agree, but unfortunately, `Android.Support.V4.App.Fragment` does not extend from Android.App.Fragment` in C# thanks to the Java bindings

```C#
namespace Android.Support.V4.App
{
    [Register("android/support/v4/app/Fragment", DoNotGenerateAcw = true)]
    public class Fragment : Java.Lang.Object, ILifecycleOwner, IJavaObject, IDisposable, IComponentCallbacks, Views.View.IOnCreateContextMenuListener
    {
        [Register(".ctor", "()V", "")]
        public Fragment();
        protected Fragment(IntPtr javaReference, JniHandleOwnership transfer);

        // ...
```

So these extension methods are required

Reply:
I noticed so for Fragments, see my comments in the issue, but this is the case fro `AppCompatActivity` though.

Reply:
Ah, gotcha. Will get on to it